packages/cli/src/utils/cli.ts

```diff
@@ -59,3 +59,8 @@ export function failWith(msg: string): never {
   console.error(msg)
   return process.exit(1)
 }
+
+export async function binaryPrompt(promptMessage: string) {
+  const resp = await cli.prompt(promptMessage)
```

Comment:
maybe add a fixed "[y/yes, n/no]" as a prompt suffix?

Reply:
i don't if oclif has it, but a good library for this kind of things is `inquirer.js`